gym/wrappers/order_enforcing.py

```diff
@@ -1,18 +1,48 @@
+"""Wrapper to enforce the order of environments."""
```

Comment:
"Enforcing the order of environments" sounds like there are multiple environments that are somehow ordered...

```suggestion
"""Wrapper to enforce the proper ordering of environment operations."""
```